### PR TITLE
Update EI section in regards to model loading within MXNet README

### PR DESCRIPTION
**Pull request by ChoiByungWook**
*Issue #, if available:*
In reference to https://github.com/aws/sagemaker-mxnet-container/pull/65#discussion_r252494747. Recommended to move documentation update from https://github.com/aws/sagemaker-mxnet-container to here.

*Description of changes:*
Add detail on loading model for EI in model loading section.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/mxnet/README.rst [hunk 1]
```diff
@@ -531,6 +531,26 @@ The following code-snippet shows an example custom ``model_fn`` implementation.
...
+        :param: model_dir The directory where model files are stored.
+            :return: a model (in this case a Gluon network)
+            """
+            net = models.get_model('resnet34_v2', ctx=mx.eia(), pretrained=False, classes=10)
+            net.load_params('%s/model.params' % model_dir, ctx=mx.eia())
+            return net
+
+The `default_model_fn <https://github.com/aws/sagemaker-mxnet-container/pull/55/files#diff-aabf018d906ed282a3c738377d19a8deR71>`__ will load and serve your model through Elastic Inference, if applicable, within the SageMaker MXNet containers.
+
+* For more information on how to enable MXNet to interact with Amazon Elastic Inference, see `Use Elastic Inference with MXNet <https://docs.aws.amazon.com/dlami/latest/devguide/tutorial-mxnet-elastic-inference.html>`__.
```
Comment (laurenyu) [inline src/sagemaker/mxnet/README.rst]: I wouldn't make this a bulleted item

File: src/sagemaker/mxnet/README.rst [hunk 2]
```diff
@@ -531,6 +531,26 @@ The following code-snippet shows an example custom ``model_fn`` implementation.
...
+
+    def model_fn(model_dir):
+        """
+        Load the gluon model in an Elastic Inference context. Called once when hosting service starts.
+        :param: model_dir The directory where model files are stored.
+            :return: a model (in this case a Gluon network)
+            """
+            net = models.get_model('resnet34_v2', ctx=mx.eia(), pretrained=False, classes=10)
+            net.load_params('%s/model.params' % model_dir, ctx=mx.eia())
+            return net
```
Comment (laurenyu) [inline src/sagemaker/mxnet/README.rst]: I realize you just copied this from above, but it'd be nice if you could fix the indentation in both

**Comment by laurenyu** [comment]
And also please change the PR title/commit message to be an imperative statement

**Comment by laurenyu** [comment]
now that #622 has been merged, these changes need to also happen in the sphinx docs

File: src/sagemaker/mxnet/README.rst (hunk 1 above)
Comment (ChoiByungWook) [reply src/sagemaker/mxnet/README.rst]: removed.

File: src/sagemaker/mxnet/README.rst (hunk 2 above)
Comment (ChoiByungWook) [reply src/sagemaker/mxnet/README.rst]: fixed. also added a header for the ToC.

File: doc/using_mxnet.rst [hunk 3]
```diff
@@ -530,12 +533,32 @@ The following code-snippet shows an example custom ``model_fn`` implementation.
...
+
+    def model_fn(model_dir):
+        """
+        Load the gluon model in an Elastic Inference context. Called once when hosting service starts.
+        :param: model_dir The directory where model files are stored.
+        :return: a model (in this case a Gluon network)
+        """
+            net = models.get_model('resnet34_v2', ctx=mx.eia(), pretrained=False, classes=10)
+            net.load_params('%s/model.params' % model_dir, ctx=mx.eia())
+            return net
```
Comment (laurenyu) [inline doc/using_mxnet.rst]: the code is also overindented

Comment (ChoiByungWook) [reply doc/using_mxnet.rst]: fixed.

**Pull request merged**